### ISSUE TITLE
property validation false negative

### ISSUE THREAD

**Issue by mehcode** [rossPatton/stylint#164]
```stylus
:root {
}
```

```
Warning: property is not valid
File: styles/index.styl
Line: 1: :root {

Stylint: 0 Errors.
Stylint: 1 Warnings.
```

---

```
$ stylint --version
Stylint version: 1.0.9
```

**Comment by rossPatton** [comment]
this should be good in the latest version
